src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -362,7 +362,9 @@ protected static void leftAlignVariant(final VariantContextBuilder builder, fina
         boolean changesInAlleles = true;
 
         final Map<Allele, byte[]> alleleBasesMap = new HashMap<>();
-        alleles.forEach(a -> alleleBasesMap.put(a, a.getBases()));
+
+        // Put each allele into the alleleBasesMap unless it is a spanning deletion
```

Comment:
Why can't a spanning deletion go in here?  Is it b/c it's a one base allele which confuses the following code (since it expects one base alleles to be SNPs)?  If so, can you please state that here so it's clear why you're making an exception -- and then mention that it gets added into the fixed allele map later?  I just want it to be clear what's going on here for the next person who needs to make a change.